annotation_ls_py/src/annotation_ls_py/server.py

```
@@ -505,65 +550,42 @@ def get_annotation_source(ls: LanguageServer, params: List[Dict]) -> Optional[Di
+		uri = query_params["textDocument"]["uri"]
+		workspace = workspace_manager.get_workspace(uri)
+		if not workspace:
+			error(f"No workspace found for {uri}")
 			return []
 
 		# 根据查询范围获取工作区列表
-		query_scope = query_params.get("scope", "current_file")  # current_file, current_workspace, current_project
-
-		if query_scope == "current_file":
-			return list_annotations(ls, params)
+		query_scope = query_params.get( "scope", "current_file")
```

Flake8 (7.2.0)

[error] whitespace after '('

(E201)